### DISCUSSION TITLE
Welcome to julia2nix Discussions!

### DISCUSSION THREAD

**Discussion by thomasjm** [codedownio/julia2nix#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by idontgetoutmuch** [comment]
Thanks for setting this up and I am now using julia2nix very happily but if I have any questions or thoughts I'll put them here rather than raising tickets :-)
